guides/developer/plugin-creators/deck-plugin.md

```diff
@@ -0,0 +1,451 @@
+   spin-rosco-6885b6df45-jqkl9         1/1     Running   0          10m
+   spin-gate-75df95744b-7zvp5          1/1     Running   0          10m
+   spin-orca-766f9bbf7b-cw9f7          1/1     Running   0          10m
+   spin-echo-9bbcd9df8-td4rt           1/1     Running   0          10m
+   spin-clouddriver-55bc94ddcc-4d7cd   1/1     Running   0          10m
+	```
+
+## Configure Minnaker for local external services
+
+Decide which Spinnaker services you want to run locally. This example uses Orca and Deck.
+
+Configure Minnaker to expect the relevant services to be external:
+
+```bash
+./minnaker/scripts/utils/external_service_setup.sh orca deck
+```
+
+Output is similar to:
+
+```bash
+Generated deploymentConfigurations[0].deploymentEnvironment.customSizing:
+spin-orca:
+  replicas: 0
+spin-deck:
+  replicas: 0
+Generated local /etc/spinnaker/.hal/default/profiles/spinnaker-local.yml:
+--------------
+services:
+  orca:
+    baseUrl: http://192.168.64.1:8083
+  deck:
+    baseUrl: http://192.168.64.1:9000
+--------------
+Place this file at '~/.spinnaker/spinnaker-local.yml' on your workstation
+--------------
+services:
+  clouddriver:
+    baseUrl: http://192.168.64.5:7002
+  redis:
+    baseUrl: http://192.168.64.5:6379
+  front50:
+    baseUrl: http://192.168.64.5:8080
+  orca:
+    host: 0.0.0.0
+  gate:
+    baseUrl: http://192.168.64.5:8084
+  deck:
+    host: 0.0.0.0
+  echo:
+    baseUrl: http://192.168.64.5:8089
+  rosco:
+    baseUrl: http://192.168.64.5:8087
+--------------
+```
+
+This script creates a `spinnaker-local.yml` on the VM that indicates the IPs where Orca and Deck are running. Furthermore, the script generates Spinnaker configuration content that you need to copy on your OSX workstation. This content tells your locally running services where the rest of the Spinnaker services are running.
+
+**Note**: `external_service_setup.sh` removes the previous configuration each time you run it.
+
+## Configure your workstation for the local services
+
+Copy the `services` section between the dotted lines in the terminal output from the executing the `external_service_setup.sh` script. Create or edit the `~/.spinnaker/spinnaker-local.yml` file on your workstation and paste the previously copied `services` snippet into it.
+
+The `spinnaker-local.yml` has this content:
+
+```yaml
+services:
+  clouddriver:
+    baseUrl: http://192.168.64.5:7002
+  redis:
+    baseUrl: http://192.168.64.5:6379
+  front50:
+    baseUrl: http://192.168.64.5:8080
+  orca:
+    host: 0.0.0.0
+  gate:
+    baseUrl: http://192.168.64.5:8084
+  deck:
+    host: 0.0.0.0
+  echo:
+    baseUrl: http://192.168.64.5:8089
+  rosco:
+    baseUrl: http://192.168.64.5:8087
+--------------
+```
+
+## Clone the plugin and required services
+
+Clone the Orca and Deck `release-1.20.x` branches that correspond to the Spinnaker 1.21.0 version you installed using Minnaker.  Then clone `pf4jStagePlugin` v1.1.14, which works with Spinnaker 1.20.6.
```

Comment:
I think "Spinnaker 1.21.0" here must be wrong?

Reply:
I thought I changed all of those. Originally started with 1.21 but ran into issues running Orca locally.